typescript/eks/cluster/index.ts

```diff
@@ -16,6 +16,7 @@ class EKSCluster extends cdk.Stack {
     });
 
     const eksCluster = new eks.Cluster(this, 'Cluster', {
+      version: eks.KubernetesVersion.V1_17,
```

Comment:
there's a version key on line 22 that should have it's value updated instead of introducing a duplicate key